dashboard/src/components/AppList/AppListItem.test.tsx

```diff
@@ -17,6 +17,7 @@ import AppListItem, { IAppListItemProps } from "./AppListItem";
 const defaultProps = {
   app: {
     name: "foo",
+    pkgDisplayName: "foo",
```

Comment:
All that was missing was this, since it's used to display the name in the span (it was blank otherwise and the span was not found).